include/type_safe/constrained_type.hpp

```diff
@@ -111,7 +112,8 @@ namespace type_safe
         /// It will invoke copy(1)/move(2) constructor followed by move assignment operator.
         /// \throws Anything thrown by the copy(1)/move(2) constructor or move assignment operator of `value_type`,
         /// or the `Verifier` if the `value` is invalid.
-        /// If the `value` is invalid, nothing will be changed.
+        /// \requires `Constraint` must be copyable.
+        /// \remarks If the `value` is invalid, nothing will be changed.
```

Comment:
Oh, alright. I thought it was like that because I think I've seen something similar in the standard.